### DISCUSSION TITLE
Actions failing without error - possibly related to Docker images

### DISCUSSION THREAD

**Discussion by dirkhh** [community/community#26490] [Actions]
<p>The repo where we encounter this is Subsurface-divelog/subsurface</p>
<p>We have half a dozen CI actions, and about 5 days ago one of them started failing without any visible error, now a second one started failing. In both cases setup/init/checkout sources run fine, but then the build step never starts.</p>
<p>Downloading the log here’s what I get:</p>
<pre><code>2020-04-20T12:53:25.6221481Z ##[section]Starting: Request a runner to run this job
2020-04-20T12:53:26.2041815Z Requesting a hosted runner in current repository's account/organization with labels: 'ubuntu-latest', require runner match: True
2020-04-20T12:53:26.3402159Z Labels matched hosted runners has been found, waiting for one of them get assigned for this job.
2020-04-20T12:53:26.3514624Z ##[section]Finishing: Request a runner to run this job
</code></pre>
<p>and nothing else. The actions are using custom docker images that have a correctly set up build environment with all dependencies (one of them is creating an Android build, the other one is creating an AppImage using a highly customized Ubuntu 14.04 as base).</p>
<p>Is there a way to get better error messages? Any suggestions how to fix this?</p>

**Comment by BrightRan** [comment, marked as answer]
<p><a class="mention" href="/dirkhh">@dirkhh</a> ,</p>
<p>I checked your repository, you seem have not enabled <a href="https://help.github.com/en/actions/configuring-and-managing-workflows/managing-a-workflow-run#enabling-debug-logging" rel="nofollow noopener"><strong>debug logging</strong></a> in this repository. Maybe you can try enabling debug logging to see if you can get more detailed debug logs for the failed steps.</p>
<p>To enable debug logging, you need to add " <strong>ACTIONS_RUNNER_DEBUG</strong>" and " <strong>ACTIONS_STEP_DEBUG</strong>" as secrets in the repository and set their values to be " <strong>true</strong>". After that, you can re-run the failed workflow run or trigger a new run to view the debug logs.</p>
<p>In addition, I’m not clear which workflows have the problems as you reported. If possible, please share the YAML files of the failed workflows with us, so that we can check the detailed configurations in your workflows to analyze the root cause.</p>

**Comment by dirkhh** [comment]
<p>I have enabled these options, and like every good HeisenBug of course now the action appears to be running fine.</p>
<p>For the record, the one that had been failing was the Android one - and yesterday with the same symptoms the Ubuntu 14.04/AppImage one failed as well.</p>
<p>I will observe this through a few more Pull Request / Push triggered action runs…</p>
<p>And literally the moment that I posted the reply, the build stopped again and now I see the error.</p>
<p>Out of space</p>
